messaging.rst

```diff
@@ -85,27 +85,80 @@ Messages
 Locked Transfer
 -----------------
 
-Cancellable and expirable :term:`transfer`. Sent by a node when a transfer is being initiated, this message adds a new lock to the list of pending locks from the sending participant node.
+A LockedTransfer is a message used to reserve tokens for a new mediated transfer.
 
-Invariants
-^^^^^^^^^^
+Preconditions
+^^^^^^^^^^^^^
 
-Only valid if all the following hold:
+For a ``LockedTransfer`` to be considered valid the following conditions have to be true.
 
 - There is a channel which matches the given :term:`chain id`, :term:`token network` address, and :term:`channel identifier`.
```

Comment:
Where is the channel? Maybe we can get even more technical here. In TokenNetwork.sol or in the internal representation of the graph? On the other hand maybe that is too much ...

Reply:
since you use the term "corresponding" in the next bullet, maybe use "there is a channel corresponding to the given 'chain-id' ..."